gnocchi/opts.py

```diff
@@ -172,6 +172,11 @@ def list_opts():
                        "value may improve worker utilization but may also "
                        "increase load on coordination backend. Value is "
                        "capped by number of workers globally."),
+            cfg.IntOpt('janitor_batch_size',
```

Comment:
```suggestion
            cfg.IntOpt('cleanup_batch_size',
```
there's `metric_cleanup_delay` already

Reply:
fixed